substrate/client/transaction-pool/src/fork_aware_txpool/fork_aware_txpool.rs

```diff
@@ -554,6 +554,19 @@ where
 				"fatp::ready_at_light"
 			);
 			Box::new(tmp_view.pool.validated_pool().ready())
+		} else if let Some((most_recent_view, _)) = self
+			.view_store
+			.most_recent_view
+			.read()
+			.and_then(|at| self.view_store.get_view_at(at, true))
+		{
+			// Fallback for the case of not having a best view to use
+			// for getting a ready transactions set. Even if some of
+			// the ready txs provided by the most recent view are invalid
+			// or already included in the blocks, a few of them might still
+			// be valid for `at` hash, which is still better than including
+			// nothing.
```

Comment:
I'd slightly adjust wording here, `best view` may be misleading:
```suggestion
			// Fallback for the case when `at` is not on the already known fork.
			// Falls back to the most recent view, which may include txs which 
			// are invalid or already included in the blocks but can still yield a
			// partially valid ready set, which is still better than including nothing.
```